pkg/domain/infra/tunnel/images.go

```diff
@@ -306,7 +306,22 @@ func (ir *ImageEngine) Config(_ context.Context) (*config.Config, error) {
 }
 
 func (ir *ImageEngine) Build(_ context.Context, containerFiles []string, opts entities.BuildOptions) (*entities.BuildReport, error) {
-	return images.Build(ir.ClientCxt, containerFiles, opts)
+	report, err := images.Build(ir.ClientCxt, containerFiles, opts)
```

Comment:
I think this should just be done in the client, and should not be supported as a BuildOption, IE IIDFile should just be done in podman client.

Reply:
Although it would be cleaner in the client, our hands are a bit tied as `entities.BuildahOptions` is just an alias for the one from `imagebuildah` (see below). Since the buildah code is taking care of creating the file it (for the local client), we had to special case the iidfile creation in the client for the remote one. I think it may be easier to just leave the code here.

```golang 
// BuildOptions describe the options for building container images.
type BuildOptions struct {                                         
        imagebuildah.BuildOptions                                  
}                                                                  
```